src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/UpdateSentenceDateService.kt

```diff
@@ -108,39 +110,56 @@ class UpdateSentenceDateService(
-            "${licenceEntity.forename} ${licenceEntity.surname}",
-            licenceEntity.crn,
-            mapOf(
-              "Release date has changed to ${updatedLicenceEntity.licenceStartDate?.format(dateFormat)}" to sentenceChanges.lsdChanged,
-              "Licence end date has changed to ${updatedLicenceEntity.licenceExpiryDate?.format(dateFormat)}" to sentenceChanges.ledChanged,
-              "Sentence end date has changed to ${updatedLicenceEntity.sentenceEndDate?.format(dateFormat)}" to sentenceChanges.sedChanged,
-              "Top up supervision start date has changed to ${
-                updatedLicenceEntity.topupSupervisionStartDate?.format(
-                  dateFormat,
-                )
-              }" to sentenceChanges.tussdChanged,
-              "Top up supervision end date has changed to ${
-                updatedLicenceEntity.topupSupervisionExpiryDate?.format(
-                  dateFormat,
-                )
-              }" to sentenceChanges.tusedChanged,
-              "Post recall release date has changed to ${
-                updatedLicenceEntity.postRecallReleaseDate?.format(
-                  dateFormat,
-                )
-              }" to sentenceChanges.prrdChanged,
-            ),
-          )
-        }
+    val notifyCom = updatedLicenceEntity is HdcLicence || isNotApprovedForHdc(updatedLicenceEntity)
+    if (!notifyCom) {
+      log.info("Not notifying COM as now approved for HDC for $licenceId")
+      return
     }
+    log.info("Notifying COM ${licenceEntity.responsibleCom?.email} of date change event for $licenceId")
+    notifyService.sendDatesChangedEmail(
+      licenceId.toString(),
+      licenceEntity.responsibleCom?.email,
+      "${licenceEntity.responsibleCom?.firstName} ${licenceEntity.responsibleCom?.lastName}",
+      "${licenceEntity.forename} ${licenceEntity.surname}",
+      licenceEntity.crn,
+      mapOf(
+        "Release date has changed to ${updatedLicenceEntity.licenceStartDate?.format(dateFormat)}" to sentenceChanges.lsdChanged,
+        "Licence end date has changed to ${updatedLicenceEntity.licenceExpiryDate?.format(dateFormat)}" to sentenceChanges.ledChanged,
+        "Sentence end date has changed to ${updatedLicenceEntity.sentenceEndDate?.format(dateFormat)}" to sentenceChanges.sedChanged,
+        "Top up supervision start date has changed to ${
+          updatedLicenceEntity.topupSupervisionStartDate?.format(
+            dateFormat,
+          )
+        }" to sentenceChanges.tussdChanged,
+        "Top up supervision end date has changed to ${
+          updatedLicenceEntity.topupSupervisionExpiryDate?.format(
+            dateFormat,
+          )
+        }" to sentenceChanges.tusedChanged,
+        "Post recall release date has changed to ${
+          updatedLicenceEntity.postRecallReleaseDate?.format(
+            dateFormat,
+          )
+        }" to sentenceChanges.prrdChanged,
+        "HDC actual date has changed to ${
```

Comment:
Previously we'd consider a HDC dates change a material change that requires a notification but then didn't include the HDC specific dates that had changed in the email, added those